README.md

```diff
@@ -69,21 +69,21 @@ Setup Auto-DL:
 
```

Comment:
can you also add 🐳 emoji for docker and 👷 emoji for bare metal way please. And also just pull the main branch we have had some changes on the ReadMe.

Reply:
Can I make a new Pull Request or just edit this one?

Reply:
You can just edit this.